product_to_bizerba_scale/data/ir_cron.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<!--
+    Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
+    @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
```

Comment:
i think it's better to remove the copyright from the xml file

Reply:
done :)